module/herosystem6e.mjs

```diff
@@ -131,17 +136,17 @@ Hooks.once("init", async function () {
     Handlebars.registerHelper("is_active_segment", function (actives, index) {
         return actives?.[index];
     });
-
+    let templatePaths = [
```

Comment:
minor nit: `let` -> `const`

Reply:
fixed in adbd48101ca467ba52776f76945fe58545df20c6

Reply:
lgtm